### PR TITLE
Form validation branch

### PR DESCRIPTION
**Pull request by David-Kasilia**
### **To Highlight**

- Created form validation branch
- Used if else statements to check for forms validations.
- Used regex to check for the correct email format.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -94,4 +94,135 @@ function closing() {
...
+    } else {
+      errorMobileMessage('nameError', '');
+     nameError = false;
+    }
+  }
+
+  if (mobileEmail == '') {
+    errorMobileMessage('emailError', "*Please enter your email address")
+  } else {
+    const emailRegex = /^[a-zA-Z]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$/;
```
Comment (1GORDON) [inline script.js:199]: Hello team ✋

- [ ] [Optional] Please kindly check your validation approach. I, unfortunately, can't seem to submit my message on both the desktop and mobile forms. I can see a validation error on the desktop form but I can't see the same error on the mobile form. 
- Mobile form 
mobile validation

- Desktop form
email validation

File: stylesheet/style.css [hunk 2]
```diff
@@ -587,13 +587,13 @@ button:hover {
...
+/* input:hover {
   transform: translate(2.5rem, 2ex);
-}
+} */
 
-textarea:hover {
+/* textarea:hover {
   transform: translate(2.5rem, 2ex);
-}
+} */
```
Comment (1GORDON) [inline stylesheet/style.css:596]: Great job team 🤝

- [ ] [Optional] Kindly remove commented code from your style sheet. This is not regarded as a good practice.

**Review by 1GORDON — APPROVED** [review]
## Status: APPROVED 🟢 
## Hi @David-Kasilia & @DJ-MrJay ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Optional suggestions

- [ ] [Optional] You may please make your design responsive mostly the desktop version seems to be non-responsive.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 1 above)
Comment (David-Kasilia) [reply script.js:199]: Thank you for this review the issue was on the regex validation and now it's fixed the form can be submitted.

**Comment by David-Kasilia** [comment]
Hello Code reviewer, My dashboard had an issue I just resent the review so that I may get a chance to submit the correct link to my submission. I appreciate your help. Thank you in advance.

**Review by leonardodiasb — COMMENTED** [review]
# INVALID ❌ ❌ ❌ 

Hi @David-Kasilia, 👋

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ❌
 - You have submitted a Pull Request that is already approved.
 
 - I submitted a ZenDesk ticket informing the issue with your dashboard and the dev team will check that and approve your project if everything is alright.

Cheers and Happy coding!👏👏👏

✅ Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_⚠️ Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

----
_Invalid Code Review Request does not count into the code reviews limit._

**Pull request merged**